tests/Operation/AggregateFunctionalTest.php

```diff
@@ -109,6 +110,46 @@ public function testTypeMapOption(array $typeMap = null, array $expectedDocument
         $this->assertEquals($expectedDocuments, $results);
     }
 
+    public function testExplainOption()
+    {
+        $this->createFixtures(3);
+
+        $pipeline = [['$match' => ['_id' => ['$ne' => 2]]]];
+        $operation = new Aggregate($this->getDatabaseName(), $this->getCollectionName(), $pipeline, ['explain' => true, 'typeMap' => ['root' => 'array']]);
+        $results = iterator_to_array($operation->execute($this->getPrimaryServer()));
+
+        $this->assertCount(1, $results);
+        $this->assertArrayHasKey('stages', $results[0]);
+    }
+
+    public function testExplainOptionWithWriteConcern()
+    {
+        if (version_compare($this->getServerVersion(), '3.4.0', '<')) {
+            $this->markTestSkipped('Sessions are not supported');
+        }
```

Comment:
This skip doesn't look relevant. Missed that from an earlier version of the PR.

Reply:
This skip is relevant because, according to the [documentation for aggregate](https://docs.mongodb.com/php-library/master/reference/method/MongoDBCollection-aggregate/index.html), writeConcern "is not supported for server versions prior to 3.4 and will result in an exception at execution time if used." I forgot to change the message, which I have now corrected.